docs/en/api-reference/system/ulp-risc-v.rst

```diff
@@ -162,7 +172,7 @@ To access the ULP RISC-V program variables from the main program, the generated
 .. note::
 
     Variables declared in the global scope of the ULP RISC-V program reside in either the ``.bss`` or ``.data`` section of the binary. These sections are initialized when the ULP RISC-V binary is loaded and executed. Accessing these variables from the main program on the main CPU before the first ULP RISC-V run may result in undefined behavior.
-
+    The ``ulp_`` prefix is the default value. You can specify the prefix to use with ``ulp_embed_binary`` to avoid name collusions for multiple ULP program.
```

Comment:
Nitpick:

```suggestion
    The ``ulp_`` prefix is the default value. You can specify the prefix to use with ``ulp_embed_binary`` to avoid name collisions for multiple ULP programs.
```